time stamp
uint16 source
uint16 type
int16[] data
